include/drivers/uart.h

```diff
@@ -157,13 +157,14 @@ enum uart_rx_stop_reason {
 	UART_BREAK = (1 << 3),
 };
 
-/** @brief Backward compatibility defines, deprecated */
-#define UART_ERROR_BREAK UART_BREAK
 #define LINE_CTRL_BAUD_RATE (1 << 0)
```

Comment:
Why is this not converted to `UART_ LINE_CTRL_BAUD_RATE`?

Reply:
Because I was too focused on the deprecation problem.  Updated to put this into the enum and add the deprecated alias.